app/src/main/kotlin/com/aliucord/manager/ui/viewmodel/InstallViewModel.kt

```diff
@@ -120,13 +122,14 @@ class InstallViewModel(
         }
 
         // Download the locale split
-        val localeApkFile = step(localeApkDl, StepCategory.APK_DL) {
+        val localeApkFile = step(Steps.DL_LANG_APK, StepCategory.APK_DL) {
             externalCacheDir.resolve("config.en-${supportedVersion}.apk").also { file ->
-                if (!file.exists())
-                    downloadManager.downloadSplit(
-                        version = supportedVersion,
-                        split = "config.en"
-                    )
+                if (file.exists()) {
+                    steps[Steps.DL_LANG_APK] = steps[Steps.DL_LANG_APK]!!.copy(cached = true)
```

Comment:
Can't this just be a direct assign
or does compose not track that

Reply:
it doesn't

Reply:
death